content/decadal/1_forecasts.md

```diff
@@ -0,0 +1,22 @@
+---
+tag: Forecasts
+title: Decadal climate predictions in EUCP
+img: lead_center.png
+caption: The Lead Centre for Annual-to-Decadal Climate Prediction
```

Comment:
This image *is* not the Lead Centre, is it? And perhaps link to the source.

Reply:
Thanks for spotting this. Forgot to replace it. Now it is the correct figure, with the link to the source.